### PR TITLE
Add blubber pipeline

### PR DESCRIPTION
**Pull request by gmodena**
Add a blubber pipeline to generate service `Dockerfile`.

For example, build and run a `development` variant of a service with:
```
blubber .pipeline/blubber.yaml development > Dockerfile
docker build -t service-scaffold-golang .
docker run -p 8000:8000  service-scaffold-golang
```

Connect to `http://localhost:8000/healthz`.

### CODE REVIEW

File: .pipeline/blubber.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+runs:
+  environment:
+    APP_BASE_PATH: /srv/service
+
+variants:
+  build:
+    apt: { packages: [build-essential] }
+    copies: [local]
+    entrypoint: [make, build]
+    runs: { insecurely: false }
```
Comment (nikkhn) [inline .pipeline/blubber.yaml]: I _think_ we can omit this line, i believe it defaults to false

File: .pipeline/blubber.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  test:
+    includes: [build]
+    apt: { packages: [ca-certificates] }
+    entrypoint: [make, test]
+  prep:
+    includes: [build]
+  production:
+    copies: [prep]
+    entrypoint: [make, run]
+    runs: { environment: { APP_CONFIG_PATH: ./config.prod.yaml } }
```
Comment (gmodena) [inline .pipeline/blubber.yaml]: This is a placeholder for future config files.

File: .pipeline/blubber.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  build:
+    apt: { packages: [build-essential] }
+    copies: [local]
+    entrypoint: [make, build]
+    runs: { insecurely: false }
+  development:
+    includes: [build]
+    apt: { packages: [ca-certificates] }
+    entrypoint: [make, run]
+    runs: { environment: { APP_CONFIG_PATH: ./config.dev.yaml } }
```
Comment (gmodena) [inline .pipeline/blubber.yaml]: This is a placeholder for config files.

File: .pipeline/blubber.yaml (hunk 2 above)
Comment (nikkhn) [inline .pipeline/blubber.yaml]: i think APP_CONFIG_PATH was specific to service-runner which was part of the the node template, i dont _think_ we need it !

Comment (gmodena) [reply .pipeline/blubber.yaml]: @nikkhn ack. I was not sure if APP_CONFIG_PATH was the canonical way we name things in blubber pipelines (iirc we have the same convention in `similar-users`)

File: .pipeline/blubber.yaml (hunk 1 above)
Comment (gmodena) [reply .pipeline/blubber.yaml]: Ack. I was not sure about that by reading the doc. Should have check the code. Thanks!

**Pull request merged**